src/promptflow-devkit/promptflow/_sdk/_service/apis/line_run.py

```diff
@@ -41,7 +42,7 @@ def _parse_string_list(value: typing.Optional[str]) -> typing.Optional[typing.Li
     def from_request() -> "ListLineRunParser":
         args = list_line_run_parser.parse_args()
         return ListLineRunParser(
-            session_id=args.session,
+            collection=args.collection or args.session,
```

Comment:
if args.collection args.session both None, default to "default"?

Reply:
good catch, but maybe not here; this logic shall be better implemented in trace operation?